content/release-10-15-0/devices-10-15-0-bundle/10-15-0-327.md

```diff
@@ -0,0 +1,47 @@
+---
+weight: 23
+title: Release 10.15.0.327
+layout: bundle
+---
+
+<!--10.15.0.310 - 10.15.0.327-->
+
+### Fixes
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Device Management</td>
+<td>On the <b>Services</b> tab in the device details, the filter for the "Status" column did not work properly and returned no results when a filter was applied. This is now fixed.</td>
+<td>DM-2002</td>
+</tr>
+
+<tr>
+<td>LWM2M</td>
+<td>During the LWM2M client registration, if the agent failed to find the registered device from the database due to a communication problem, an internal object related to the device managed object was automatically removed by the agent. Afterwards, LWM2M clients always failed to connect. This is now fixed and the internal object is only removed if the actual device managed object does not exist in the database.</td>
+<td>DM-2131</td>
+</tr>
+
+<tr>
+<td>LWM2M</td>
+<td>Due to a limitation of the Leshan library used by the LWM2M agent, only device registration messages were supported where the protocol version is defined to the LWM2M object. The LWM2M agent has now been extended to also accept LWM2M registration messages when the LWM2M version is attached to the LWM2M object instance.</td>
```

Comment:
```suggestion
<td>Due to a limitation of the Leshan library used by the LWM2M agent, only device registration messages were supported where the LWM2M version is attached to the LWM2M object. The LWM2M agent has now been extended to also accept LWM2M registration messages where the LWM2M version is attached to the LWM2M object instance.</td>
```

confusing still - what is the difference between an object and the object instance?
from basic programming: the object is the instance of a class

Reply:
@Oezge-Akin Can you please have one more look?